README.md

```diff
@@ -63,3 +63,168 @@ node create-bundle.js
 ```
 The generated files should be persisted in the VCS, or generated before the application is executed/packaged.
 
+## Remote Dev demo walk through
+
+### Requirements
+
+- An Openshift cluster. The Red Hat developer sandbox is good enough to use. See https://developers.redhat.com/developer-sandbox
+- [Jbang](https://www.jbang.dev/) is required to deploy the RabbitMQ and PostgreSQL services
+- Java 17
+
+### Connect to the cluster
+Once the Openshift cluster is up and running, you can connect your local machine to the remote cluster using the command provided in the Openshift Developer Console: Click on the question mark icon, `Command line tools`, `Copy login command`. 
+
+![Openshift copy login command](./assets/openshift-copy-login.png)
+
+In my case, the proposed command to login was something similar to:
+
+    $ oc login --token=xxxxxxxxxxx --server=https://api.sandbox.x8i5.p1.openshiftapps.com:6443
+
+If you don't have the `oc` command, you can download it from the `Command line tools` page.
```

Comment:
Relates to: https://github.com/eclipse/jkube/issues/1415

This is a pain point we need to solve to be able to onboard users (both to OpenShift and JKube) easily